README updated (#2)
microsoft/makecode-minecraft-music: README.md

## MakeCode for Minecraft Education Extension

> This repository is an approved **extension** in Makecode for Minecraft Education. If you would like to make changes or add features to this extension, feel free to open a PR or file an issue!

Reviewer: ```suggestion
This repository is an approved **extension** in Microsoft MakeCode for Minecraft Education. If you would like to make changes or add features to this extension, feel free to open a PR or file an issue!
```

## Overview

<img src='./icon.png'>

> This extension is your music stop for Minecraft Education. Using the `/playsound` command underneath, the music blocks can help you hear popular Minecraft music discs like pigstep or make fun compositions with instruments or mob sounds. 

Reviewer: ```suggestion
This extension add music blocks to Microsoft MakeCode for Minecraft Education. Using the `/playsound` command underneath, the music blocks can help you hear popular Minecraft music discs like pigstep or make fun compositions with instruments or mob sounds. 
```
